docker/sandbox/site.conf

```diff
@@ -80,5 +85,9 @@ server {
       rewrite ^/(.*)$ /index.php?title=$1 break;
     }
 
+    # SUS-5863 Short URLs as http://www.wikia.com/Special:Version should not be processed by redirect-canonical.php
```

Comment:
Actually, this one should be handled by the above rule, but for some reason is not.